react/src/pages/ServingListPage.tsx

```diff
@@ -360,9 +359,7 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
                   ? {
                       title: t('modelService.CreatedUser'),
                       dataIndex: 'created_user',
-                      render: (created_user) => (
-                        <EndpointCreatedUserText userId={created_user} />
-                      ),
+                      render: (created_user) => created_user,
                     }
                   : {},
```

Comment:
When `is_admin` is `false, `{}` is added to the array. It's not a good idea even `Table` doesn't show it. 
One of the solution is
```js
onst array = [
  1,
  2,
  ...(is_admin ? [3] : []),
  4
];
```

Reply:
Oh.. I didn't know that!😳